bundles/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/handler/BigAssFanHandler.java

```diff
@@ -660,7 +669,7 @@ private String readMessage() {
             return message;
         }
 
-        private void processMessage(String incomingMessage) {
+        private void processMessage(@Nullable String incomingMessage) {
```

Comment:
Consider not calling method with null argument.

Reply:
I rather leave it as is. I would need to move the nullness check from here to the caller. The same, but different.

Reply:
The only difference is that it would be cleaner not calling a method without needed arguments in order to do nothing, since this could hide bugs. It could then even throw an [IllegalArgumentException](https://docs.oracle.com/javase/7/docs/api/java/lang/IllegalArgumentException.html) to make it fail fast and prevent introduction of bugs. However, this is out of scope here, just wanted to mention this difference.